WordPress/src/main/res/layout/actionable_empty_view.xml

```diff
@@ -11,6 +11,7 @@
         android:layout_width="match_parent" >
 
         <LinearLayout
+            android:importantForAccessibility="yes"
```

Comment:
This is nice. I noticed that the user can still select the larger view behind this:

<img src="https://user-images.githubusercontent.com/198826/68507523-54edb280-0229-11ea-862b-573dbcd21655.gif" width="320">

When hovering over that view, it says “List”. 😕Is it possible to cover the whole view instead?

Reply:
I was thinking to just disable the "List" part.

Reply:
When you say cover the whole view instead do you mean that when the view is empty hovering anywhere on it will initiate the announcement?

Reply:
More like the ”List” part would say the ”No posts saved — yet! Tap...”. If we do this, it's probably easier to _find_ the message because the finger doesn't need to travel far.

Reply:
Okay understood. It might be a bit trickier to do that because the list won't be the item that has the accessibility content so there would need to be some mechanism to trigger/propagate this from the list to the empty view. I will still look into and see.

Reply:
Thanks. I was thinking it would be possible to extend the empty view so it's the only one that you can tap. That seems to be the case in the XML. 

![image](https://user-images.githubusercontent.com/198826/68511013-2de7ae80-0232-11ea-9316-af8c33add8db.png)

But it looks like when it's running, it gets shortened? Not sure. 🤔 Anyway, thanks for taking a look at it!

Reply:
Oh, that makes sense. Will look at it from that angle. What I was able to achieve though is the disabling of the "List" focus.

Reply:
> But it looks like when it's running, it gets shortened? Not sure. 🤔 Anyway, thanks for taking a look at it!

Yes, I am wondering why this happens. I am gonna check to see because the current properties should allow it to take up the whole screen.

Reply:
I have been checking but not getting anywhere meaningful. So I will revisit another time. I think what we have now is good since we are announcing the state and then they can hover to hear it again.

Reply:
"List" is no longer announced https://github.com/wordpress-mobile/WordPress-Android/pull/10741/commits/6b080de8e39b68818e2817aa37543e95b1622a6e